test/run-test.rb

```diff
@@ -10,6 +10,11 @@
 
 $LOAD_PATH.unshift(lib_dir)
 
+unless ENV["TESTOPTS"].nil?
+  args = ENV["TESTOPTS"].split(" ")
```

Comment:
Could you use `Shellwords.shellsplit` instead to handle arguments that have one more spaces correctly?

Reply:
Yes, I don't know it. Thanks!